lib/grinder_files.dart

```diff
@@ -267,6 +268,8 @@ class FilePath {
 
   static String _coerce(arg) {
     if (arg is String) {
+      if (_sep != '/') arg = arg.replaceAll('/', _sep);
```

Comment:
So this would convert e.g. `/foo/bar` to `\foo\bar` on windows.  Should that be something like `C:\foo\bar`?  Should either grinder (or grinder users) just be using the `path` package for this?